src/Catalog.php

```diff
@@ -15,14 +15,16 @@
  * under the License.
  */
 
-require_once(dirname(__FILE__) . '/gen/killbill_catalog_attributes_simple.php');
+namespace Killbill\Client;
 
-class Killbill_Catalog extends Killbill_CatalogAttributesSimple {
+use Killbill\Client\Type\CatalogAttributes;
+
+class Catalog extends CatalogAttributes {
     private $fullCatalog;
 
     public function initialize($headers = null) {
-        $response = $this->_get(Killbill_Client::PATH_CATALOG, $headers);
-        $this->fullCatalog = json_decode($response->body);
+        $response = $this->_get(Client::PATH_CATALOG, $headers);
+        $this->fullCatalog = json_decode($response->body)[0];
```

Comment:
> Alright, so it is possible to get multiple catalogs here?

Yes, in most cases actually you would have a list of historical catalogs.

> This should probably be reflected in the PHP client in some way.

👍